google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionOptions.java

```diff
@@ -573,11 +591,21 @@ private ConnectionOptions(Builder builder) {
     if (builder.credentials == null
```

Comment:
Shall we use `this.fixedCredentials` instead of `builder.credentials` here, to maintain consistency with the other credentials check?

Reply:
Good point, done.